app/controllers/api_controller/report_results.rb

```diff
@@ -0,0 +1,24 @@
+class ApiController
+  module ReportResults
+    #
+    # Report Results Supporting Methods
+    #
+    def results_query_resource(object)
+      object.send("miq_report_results")
+    end
+
+    def show_reports
+      return show_generic(:reports) unless @req[:subcollection]
```

Comment:
2 things, only renderer.rb should be jbuilder aware, the other thing is that we do want to get result_set if someone does GET /api/reports/80/results?expand=resources, so not only when subcollection id is specified.  Let me think about this a bit more.